bagelbids/tests/data/README.md

```diff
@@ -9,7 +9,7 @@ Example inputs to the CLI
 | 3       | same as example 2                                                   | valid BIDS data dictionary, BUT: does not contain Neurobagel `"Annotations"` key | fail   |
 | 4       | valid, has additional columns not described in `.json`              | same as example 1                                                                | pass   |
 | 5       | valid, has additional unique value, not documented in `.json`       | same as example 1                                                                | fail   |
-| 6       | valid, same as example 5                                            | valid, contains `"MissingValues"` attribute for categorical variable             | pass   |
+| 6       | valid, same as example 5. hass annotation tool columns              | valid, contains `"MissingValues"` attribute for categorical variable             | pass   |
```

Comment:
Thanks! just in case you didn't know, GH let's you suggest simple changes directly as a comment: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request